pages/faq.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div id="main-content" class="text-wrapper">
+    <h1 class="text-center mb-6">FAQ with Professor Julian Goodare</h1>
+    <p class="mb-6 text-justify">
+      University of Edinburgh students, Ruby Imrie and Ellie Whitehead, sat down
+      with Professor Julian Goodare, lecturer in Scottish History, to ask him
+      some questions about the Scottish witch trials and the Survey of Scottish
+      Witchcraft. Here are his answers in the video playlist below. For more
+      FAQ, please refer to the
+      <a
+        class="underline hover:text-gray"
+        href="https://witches.hca.ed.ac.uk/faq/"
+        >Survey of Scottish Witchcraft FAQ page
+      </a>
+      or contact us using our
+      <a class="underline hover:text-gray" href="/contact">contact form.</a>
+    </p>
+    <div class="flex flex-col items-center mb-3">
+      <h2>The Scottish Witch Trials</h2>
```

Comment:
I think it might be better if the section headings were left aligned.

Reply:
done